ote/src/cljs/ote/theme/colors.cljs

```diff
@@ -1,76 +1,127 @@
+(def ^:private sand "#c89b64")
+;  Fintraffic basic colors
+(def basic-blue "#0064eb")
+(def basic-green "#25a794")
 (def basic-purple "#73468c")
+(def basic-red "#ff5a05")
+(def basic-yellow "#ffd000")
+(def basic-gray "#9696aa")
 (def basic-brown "#784b28")
+(def basic-brown sand)
 
-;; Dark colors.
+;; Dark colors
+;  semantic names defined by designers, do not use these directly
+(def ^:private orange "#ff8c00")
+(def ^:private brown "#784b28")
+;  Fintraffic dark colors
 (def dark-blue "#0034ac")
 (def dark-green "#005f61")
 (def dark-purple "#520076")
 (def dark-red "#b40000")
+(def dark-yellow orange)
 (def dark-gray "#505064")
+(def dark-brown brown)
 
+;; Light colors
+;  semantic names defined by designers, do not use these directly
+(def beige "#dcc8aa")
+;  Fintraffic light colors
+(def light-blue "#5fb4f5")
+(def light-green "#64e1b4")
+(def light-purple "#af96dc")
+(def light-red "#ff9b87")
+(def light-yellow "#fff5aa")
+(def light-gray "#cdcdd7")
+(def light-brown beige)
+
+;; Semantic colors - prefer the use of these above everything else
+;  Primary colors by usage context.
 (def primary-background-color basic-black)
 (def primary-text-color basic-white)
 (def primary-button-background-color dark-blue)
 
-(def blue-light "#3385d6")
-(def blue-dark "#0048c2")
-(def blue-darker "#0029B8")
-
-(def gray950 "#191919")
-(def gray900 "#323232")
-(def gray800 "#4b4b4b")
-(def gray750 "#646464")
-(def gray700 "#7d7d7d")
-(def gray650 "#969696")
-(def gray550 "#c8c8c8")
-(def gray450 "#dcdcdc")
-(def gray400 "#e1e1e1")
-(def gray350 "#e6e6e6")
-(def gray300 "#ebebeb")
-(def gray200 "#f0f0f0")
-(def gray100 "#f5f5f5")
-(def gray50 "#fafafa")
-
-(def green-basic "#00AA00")
-(def red-basic "#DD0000")
-(def red-dark "#CF0000")
-(def red-darker "#C10000")
-(def orange-basic "#FF8800")
-(def white-basic "white")
-(def yellow-basic "#ddcc00")
-(def purple-darker "#7000D0")
-
-(def add-color green-basic)
-(def icon-disabled gray550)
-(def icon-gray gray900)
-(def negative-button red-basic)
-(def negative-text white-basic)
-(def negative-button-hover red-dark)
-(def primary dark-blue)
-(def primary-dark blue-dark)
-(def primary-darker blue-darker)
-(def primary-text white-basic)
-(def primary-disabled gray550)
-(def primary-light blue-light)
-(def secondary gray900)
-(def progress primary)
-(def remove-color red-darker)
-(def success green-basic)
-(def warning red-basic)
-
-(def calendar-day-font gray950)
-
-(def monitor-taxi-color "rgb(0,170,187)")
-(def monitor-request-color "rgb(102,204,102)")
-(def monitor-schedule-color "rgb(153,204,0)")
-(def monitor-terminal-color "rgb(221,204,0)")
-(def monitor-rental-color "rgb(255,136,0)")
-(def monitor-parking-color "rgb(255,102,153)")
-(def monitor-brokerage-color "rgb(153,0,221)")
+; colors with just a single variant do not have the color intensity in the name
+(def accessible-blue dark-blue)
+(def accessible-green dark-green)
+(def accessible-light-purple light-purple)
```

Comment:
This line should be instead: ```
(def accessible-basic-purple basic-purple)```

`light-purple` (`"#af96bc"`) is not in the accessible colors, but `basic-purple` (`"#73468c"`) is.